mobile-sdks.md

```diff
@@ -68,7 +68,7 @@ Fidel.present(self, onCardLinkedCallback: { (card: LinkResult) in
 })
 ```
 
-If you run into any issues, [let us know in the Developer Community](https://community.fidel.uk/c/Issues-or-Questions-on-our-mobile-SDKs/) or [check issues on Github](https://github.com/FidelLimited/fidel-ios).
+If you run into any issues, [check issues on Github](https://github.com/FidelLimited/fidel-ios).
```

Comment:
`If you run into any issues, please refer to the [issues section of SDK repository](https://github.com/FidelLimited/fidel-ios)`?

The other sentence seems that if I "have issues" I should "check issues" 🤔 

Same on the Android SDK part.

Reply:
I fixed that, thank you!